### PR TITLE
docs: add keep screen alive entry

### PR DESCRIPTION
**Pull request by paaff**
## What type of PR is this?
Adding an entry to the documentation's FAQ section.

- documentation

## What this PR does / why we need it:

I've recently started using Mealie and came across the issue of not being able to find the Keep Screen Alive button. I began searching the documentation, code and discord and found that other's had mentioned why it usually didn't appear. Either not using HTTPS or not using a supported browser. That was also the case in my instance. :) 

So I thought, why not add to the FAQ in the event future new users wondered the same thing. 

## Which issue(s) this PR fixes:

No issues.

### CODE REVIEW

**Comment by boc-the-git** [comment]
I believe there's also a known failure case on iOS. I'll suggest some wording when I get the chance, if no one beats me to it.

File: docs/docs/documentation/getting-started/faq.md [hunk 1]
```diff
@@ -149,3 +149,11 @@ As to why we need a database?
 
 - **Developer Experience:** Without a database, a lot of the work to maintain your data is taken on by the developer instead of a battle-tested platform for storing data.
 - **Multi User Support:** With a solid database as backend storage for your data, Mealie can better support multi-user sites and avoid read/write access errors when multiple actions are taken at the same time.
+
+## Why is there no Keep Screen Alive button when I access a Mealie Recipe?
```
Comment (michael-genson) [inline docs/docs/documentation/getting-started/faq.md]:
```suggestion
## Why is there no "Keep Screen Alive" button when I access a recipe?
```

File: docs/docs/documentation/getting-started/faq.md [hunk 2]
```diff
@@ -149,3 +149,11 @@ As to why we need a database?
...
+You've perhaps visited the Mealie Demo and noticed that it had a Keep Screen Alive button, but it doesn't show up in your own Mealie instance.
```
Comment (michael-genson) [inline docs/docs/documentation/getting-started/faq.md]:
```suggestion
You've perhaps visited the Mealie Demo and noticed that it had a "Keep Screen Alive" button, but it doesn't show up in your own Mealie instance.
```

File: docs/docs/documentation/getting-started/faq.md [hunk 3]
```diff
@@ -149,3 +149,11 @@ As to why we need a database?
...
+There are typically two possible reasons for this.
```
Comment (michael-genson) [inline docs/docs/documentation/getting-started/faq.md]:
```suggestion
There are typically two possible reasons for this:
```

File: docs/docs/documentation/getting-started/faq.md [hunk 4]
```diff
@@ -149,3 +149,11 @@ As to why we need a database?
...
+- You're accessing your Mealie instance without using HTTPS. The Wake Lock API is only available if HTTPS is used. Read more here: https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API
```
Comment (michael-genson) [inline docs/docs/documentation/getting-started/faq.md]:
```suggestion
1. You're accessing your Mealie instance without using HTTPS. The Wake Lock API is only available if HTTPS is used. Read more here: https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API
```

File: docs/docs/documentation/getting-started/faq.md [hunk 5]
```diff
@@ -149,3 +149,11 @@ As to why we need a database?
...
+- Or, you're accessing your Mealie instance on a browser which is not supported. You can test this here: https://vueuse.org/core/useWakeLock/#demo
```
Comment (michael-genson) [inline docs/docs/documentation/getting-started/faq.md]:
```suggestion
2. You're accessing your Mealie instance on a browser which doesn't support the API. You can test this here: https://vueuse.org/core/useWakeLock/#demo
```

**Review by michael-genson — CHANGES_REQUESTED** [review]
Thanks for this! A few minor grammar/clarity things

**Review by paaff — COMMENTED** [review]
Looks good to me, definitely an improvement. :)

**Comment by paaff** [comment]
Yo @michael-genson - Kinda new in making PR's, committing the suggested changes and such. 
I hope I did it correctly, but if not, please let me know :)

**Review by michael-genson — APPROVED** [review]
Looks great, thank you!

**Pull request merged**